s3/s3_log_shipping/README.md

```diff
@@ -12,8 +19,12 @@ Import the module into your terraform
 ``` hcl
 module "s3_log_shipping" {
   source  = "github.com/alphagov/cyber-security-shared-terraform-modules//s3/s3_log_shipping"
-  sqs_arn = aws_sqs_queue.YOUR_SQS.arn
-  s3_name  = aws_s3_bucket.YOUR_BUCKET.id
+  s3_processor_lambda_role = # To be provided by the Cyber team
+  s3_processor_sqs_arn = # To be provided by the Cyber team
```

Comment:
slightly confused by this line - we have it working without using the `s3_processor_sqs_arn`, but I see you've added it as a  variable further down - it was previously working without it

Reply:
To simplify things, would it not be easier to have cyber supply the sqs arn and the lambda role, rather than having everyone who wants to connect have to ask cyber for them, since they will always be the same?